tendrl/commons/objects/cluster/atoms/import_cluster/__init__.py

```diff
@@ -151,6 +151,9 @@ def run(self):
                                 }
                             )
                         )
+                    else:
+                        NS.node_context.tags += ['provisioner/%s' % integration_id]
```

Comment:
Should not set this tag for ceph, since ceph-installer is pre-installed by the user.on any cluster node and tendrl detects the ceph-installer service and tags that node "provisioner/$integration_id"

Please add a todo here to handle this case here